### PR TITLE
Delete or organize non-core files

### PR DESCRIPTION
**Pull request by MrCreosote**
This has needed to be done for a long time...

Removed no longer useful files, consolidated other files that aren't
core code under `performanceanddevelopment`. Hopefully this will make
the codebase a bit easier to understand

### CODE REVIEW

File: hooks/build [hunk 1]
```diff
@@ -1,16 +0,0 @@
-#!/bin/bash
```
Comment (MrCreosote) [inline hooks/build:1]: dockerhub build cruft

File: Makefile [hunk 2]
```diff
@@ -69,10 +69,7 @@ compile-typespec:
 		$(SERVICE).spec
 	rm lib/biokbase/workspace/authclient.py
 
-test: test-client test-service
-
-test-client: test-service
-	$(ANT) test_client_import
```
Comment (MrCreosote) [inline Makefile:75]: Testing that the client built correctly made sense back when the Workspace was the first and only Java SDK service. Now it's a waste of time

File: src/AllWorkspaceTestsRunner.java [hunk 3]
```diff
@@ -1,76 +0,0 @@
-import java.io.File;
```
Comment (MrCreosote) [inline src/AllWorkspaceTestsRunner.java:1]: Unused, you can do the same thing with patterns in ant anyway (and presumably Gradle, etc.)

**Review by bio-boris — APPROVED** [review]

**Comment by MrCreosote** [comment]
LGTM and the docker build were already successful for this PR, changing the base to develop caused them to be rerun

**Pull request merged**